pkg/network/setup/netstat_test.go

```diff
@@ -231,132 +241,128 @@ var _ = Describe("netstat", func() {
 			},
 		}
 
-		domain := &api.Domain{
-			Spec: api.DomainSpec{Devices: api.Devices{Interfaces: []api.Interface{
-				newDomainSpecIface(primaryNetworkName, origMAC),
-			}}},
-			Status: api.DomainStatus{Interfaces: []api.InterfaceStatus{
-				newDomainStatusIface(primaryNetworkName, []string{newGaIPv4, newGaIPv6}, origMAC),
-			}},
-		}
+		setup.addGuestAgentInterfaces(
+			newDomainStatusIface(primaryNetworkName, []string{newGaIPv4, newGaIPv6}, origMAC, ""),
+		)
 
-		netStat.UpdateStatus(vmi, domain)
+		setup.NetStat.UpdateStatus(setup.Vmi, setup.Domain)
 
-		Expect(vmi.Status.Interfaces).To(Equal([]v1.VirtualMachineInstanceNetworkInterface{
+		Expect(setup.Vmi.Status.Interfaces).To(Equal([]v1.VirtualMachineInstanceNetworkInterface{
 			newVMIStatusIface(primaryNetworkName, []string{newGaIPv4, newGaIPv6}, origMAC, ""),
 		}), "the pod IP/s should be reported in the status")
 	})
 
-	It("should add a new interface based on the domain spec", func() {
+	It("should report SR-IOV interface when no guest-agent is active", func() {
```

Comment:
Please consider adding a test with 2 interfaces (without guest agent)
1. Non SR-IOV
2. SR-IOV

Because the behavior would be different, as the main loop of `UpdateStatus` would override the interfaces added by the snippet 
https://github.com/kubevirt/kubevirt/pull/6951/files#diff-c2b23c7cf6d14c6f6a9e12f1a63d6c8004ead588e9d4d6a369aeea15701ca086R79

Reply:
Done